doc/setup/tomcat.md

```diff
@@ -48,6 +48,7 @@ sudo keytool -importkeystore \
     -destkeystore /etc/tomcat6/keystore
 ```
 
+The srckeystore is "changeit" by default, and should be modified in /etc/default/cacerts.
```

Comment:
keytool asks for 2 passwords : one for srckeystore and the other for destkeystore.